src/language-js/utils.js

```diff
@@ -1304,11 +1301,7 @@ function hasComment(node, flags, fn) {
     return false;
   }
   const test = getCommentTestFunction(flags, fn);
-  return test
-    ? node.comments.some((comment, index, comments) =>
-        test(comment, index, comments)
-      )
-    : true;
+  return test ? node.comments.some(test) : true;
```

Comment:
Can we revert this change? https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-callback-reference.md

Reply:
I don't approve that rule at all. The issue that it's supposed to prevent happens too seldom. With TS checks, even if the type coverage is as poor as we have it in Prettier, it happens even more seldom. But the added verbosity is ugly.